docker/README.md

```diff
@@ -127,81 +132,79 @@ The haproxy listens on ports 5000 (connects to the coordinator primary) and 5001
+    b8924834836e   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-work1-1
+    b45f8daaa173   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-coord3
+    2ae2ee63737a   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-work1-2
+    6f3645abd0b3   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-etcd2
+    4ef625cca07a   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-etcd3
+    8e4a50e542fa   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-coord1
+    da7ae11caec4   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-coord2
+    43998188468b   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes   0.0.0.0:5000-5001->5000-5001/tcp, :::5000-5001->5000-5001/tcp   demo-haproxy
+    cf69f12dc472   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-work2-1
+    ea0e32ca68f5   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-work2-2
+    52376efe98d2   patroni-citus   "/bin/sh /entrypoint…"   4 minutes ago   Up 4 minutes                                                                   demo-etcd1
+
 
     $ docker logs demo-coord1
-    2023-01-05 15:09:31,295 INFO: Selected new etcd server http://172.27.0.4:2379
-    2023-01-05 15:09:31,388 INFO: Lock owner: None; I am coord1
-    2023-01-05 15:09:31,501 INFO: trying to bootstrap a new cluster
-    ...
-    2023-01-05 15:09:45,096 INFO: postmaster pid=39
+    2023-11-15 15:53:01,467 INFO: Selected new etcd server http://172.20.0.5:2379
+    2023-11-15 15:53:01,504 INFO: No PostgreSQL configuration items changed, nothing to reload.
+    2023-11-15 15:53:01,566 INFO: Lock owner: None; I am coord1
+    2023-11-15 15:53:01,609 INFO: waiting for leader to bootstrap
+    2023-11-15 15:53:05,160 INFO: Lock owner: coord3; I am coord1
+    2023-11-15 15:53:05,160 INFO: trying to bootstrap from leader 'coord3'
+    2023-11-15 15:53:05,308 INFO: Lock owner: coord3; I am coord1
+    2023-11-15 15:53:05,358 INFO: bootstrap from leader 'coord3' in progress
+    2023-11-15 15:53:06,754 INFO: replica has been created using basebackup
+    2023-11-15 15:53:06,755 INFO: bootstrapped from leader 'coord3'
```

Comment:
The same story as with https://github.com/zalando/patroni/pull/2946/files#diff-da8fcbe728a9172b578e5d754f8e2df214c658c4321f610e63dd68bea828ab49R48-R53

Reply:
Did for citus too and now its initializing a new cluster on `coord1`